src/mcpbr/benchmarks/supermodel/endpoints/base.py

```
@@ -0,0 +1,93 @@
+        return response
+
+    def scope_prompt(self, prompt: str, scope_prefix: str | None) -> str:
+        """Append scope context to a prompt if a scope_prefix is set."""
+        if scope_prefix:
+            return prompt + f"\n\nNote: Focus your analysis on the {scope_prefix} directory."
+        return prompt
+
+    @staticmethod
+    def get_pr_diff(repo: str, pr_number: int) -> str:
+        """Fetch a PR diff from GitHub using the gh CLI."""
+        result = subprocess.run(
```

Ruff (0.15.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)